### ISSUE TITLE
Youtube Link Canonicalizer signature on unrelated edits

### ISSUE THREAD

**Issue by jesus2099** [zabe40/musicbrainz-userscripts#7]
It is not your userscript fault, editors should submit only [this YouTube edit](https://musicbrainz.org/edit/108548071) **alone**.

But to avoid your userscript signature showing on [unrelated edits](https://musicbrainz.org/search/edits?auto_edit_filter=&order=desc&negation=0&combinator=and&conditions.1.field=id&conditions.1.operator=BETWEEN&conditions.1.args.0=108548069&conditions.1.args.1=108548071&field=Please+choose+a+condition), maybe you could consider opening the URL edit page instead of the artist edit page?

**Comment by zabe40** [comment]
Getting the MBID of the URL is proving harder than expected. Right now I'm using the official API to search for it, but I only get the right response about half the time, and the other half is just random unrelated URLs. Heres an [example API query](https://musicbrainz.org/ws/2/url?query=https%3A%2F%2Fwww.youtube.com%2Fannapantsu&url=https%3A%2F%2Fwww.youtube.com%2Fannapantsu&relationtype=youtube&targetid=8414a0c5-ad21-4f19-a85e-ffc3f2fea09c&targettype=artist&limit=25&fmt=json) to search for [this URL](https://www.youtube.com/annapantsu). Do you know if there's something wrong with my query, or is this just a quirk/bug of the API?

**Comment by jesus2099** [comment]
What I did in ALL LINKS is that I queried URL relationships from current entity.
https://github.com/jesus2099/konami-command/blob/03b5bb0026fa32452a6dd89e42a7e99f1c2d843a/mb_ALL-LINKS.user.js#L893

Then I don't remember exactly what I did but if I find that the URL was there in the page, I add its edit button. Which means I know its MBID.
https://github.com/jesus2099/konami-command/blob/03b5bb0026fa32452a6dd89e42a7e99f1c2d843a/mb_ALL-LINKS.user.js#L1114
